### PR TITLE
Ansible fixes

### PR DESCRIPTION
**Pull request by erikh**
ansible fixes for volplugin

@mapuri

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -51,7 +51,8 @@ ansible_provision = proc do |ansible|
     fsid: '4a158d27-f750-41d5-9e7f-26ce4c9d2d45',
     monitor_secret: 'AQAWqilTCDh7CBAAawXt6kyTgLFCxSvJhTEmuw==',
     journal_size: 100,
-    monitor_interface: "enp0s8",
+    control_interface: "enp0s8",
+    netplugin_if: "enp0s8",
```
Comment (mapuri) [inline Vagrantfile]: @erikh do you want to set variable now or wait until we bring back `contiv_network` in volplugin host-group?

**Comment by mapuri** [comment]
LGTM

File: Vagrantfile (hunk 1 above)
Comment (erikh) [inline Vagrantfile]: yeah it's fine. I wont' have to deal with it at that point then.

Comment (mapuri) [inline Vagrantfile]: sounds good. Yes, it works unless I decide to change the variable again :D

**Pull request merged**